### DISCUSSION TITLE
bootbox with devexpress BootstrapChart not working

### DISCUSSION THREAD

**Discussion by gmarrero777** [makeusabrew/bootbox#845] [General]
Hello, for my bootbox it is the best free dialog library for Bootstrap but I have a problem and that is that when invoking bootbox.confirm or bootbox.alert when the Devexpress BootstrapChart component is also displayed, what happens is a postcallback, however this does not occurs when the BootstrapChart component is not present.
There is no response from Devexpress.com support.

**Comment by tiesont** [comment]
I'm not entirely sure what you're asking, here. Going to need more details.
